src/components/enterprise-page/EnterprisePage.jsx

```diff
@@ -63,11 +63,6 @@ export default function EnterprisePage({ children }) {
               href: `${process.env.LMS_BASE_URL}/account/settings`,
               content: 'Account Settings',
             },
-            {
-              type: 'item',
-              href: process.env.ORDER_HISTORY_URL,
-              content: 'Order History',
-            },
```

Comment:
Are we intentionally removing this?

Reply:
Yeah, this was intentional. It was from a conversation with Joe that viewing "Order History" doesn't really make sense for Enterprise learners. I plan to do a quick follow-up with him just to re-confirm, though.

Reply:
Got confirmation from Joe about removing the link :)